test/python/circuit/test_commutation_checker.py

```diff
@@ -409,6 +410,16 @@ def test_annotated_operations_commute_with_unannotated(self):
         self.assertTrue(scc.commute(op1, [0, 1], [], op3, [1], []))
         self.assertTrue(scc.commute(op2, [1], [], op3, [1], []))
 
+    def test_utf8_gate_names(self):
+        """Check compatibility of non-ascii quantum gate names."""
+        g0 = RXXGate(1.234).to_mutable()
```

Comment:
FWIW, you don't need the `to_mutable()` here because `RXXGate` is always mutable (I have a branch experimenting with making it a singleton here: https://github.com/mtreinish/qiskit-core/commit/e2ff9fff93529440173c6049e184c2f224256ec1 but the performance tradeoffs aren't really worth it from what I've tested) so this just creates a copy. But this doesn't hurt anything in a test and it's more explicit.